website/source/api/index.html.md

```diff
@@ -77,6 +77,47 @@ to the supplied maximum `wait` time to spread out the wake up time of any
 concurrent requests. This adds up to `wait / 16` additional time to the maximum
 duration.
 
+### Implementation Details
+
+While the mechanism is relatively simple to work with, there are a few edge 
+cases that must be handled correctly.
+ * **Reset the index if it goes backwards**. While indexes in general are 
```

Comment:
```suggestion
 * **Reset the index if it decreases below zero**. While indexes in general are 
```

Reply:
That's not quite the right meaning - it will never be less than zero but might "go backwards" in that you'll get a smaller number than you had before.

Can you think of a better way of saying that? Technically it would be more accurate to say "breaks monotonicity" but that's not better for understanding.